onnxruntime/python/tools/transformers/README.md

```diff
@@ -21,6 +21,15 @@ After it is installed, you can use command like the following to optimize model:
 python -m onnxruntime_tools.transformers.optimizer --input gpt2.onnx --output gpt2_opt.onnx --model_type gpt2
```

Comment:
python -m onnxruntime_tools.**optimizer_cli** --input gpt2.onnx --output gpt2_opt.onnx --model_type gpt2

A pythonic way is to have _cli or .cli indicate this python file was invoked by the command line.